osu.Game.Tests/Online/Chat/MessageNotifierTest.cs

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using NUnit.Framework;
+using osu.Game.Online.Chat;
+
+namespace osu.Game.Tests.Online.Chat
+{
+    [TestFixture]
+    public class TestCheckUsername
```

Comment:
test suite should match file name.
```suggestion
    public class MessageNotifierTest
```